script.artwork.downloader/lib/old_apply_filters.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/python
```

Comment:
Is this file ever used? I don't think so, so I would suggest to just not include it?

Reply:
I don't familiar with this file as I wrote you.
I will check it, if it actually never used maybe I will remove it.